src/Altinn.App.Core/Internal/Expressions/ExpressionEvaluator.cs

```diff
@@ -485,39 +485,19 @@ bool ab
             return bvalue ? "true" : "false";
         }
 
-        if (value is string svalue)
+        return value switch
         {
             // Special case for "TruE" to be equal to true
-            if ("true".Equals(svalue, StringComparison.OrdinalIgnoreCase))
-            {
-                return "true";
-            }
-            else if ("false".Equals(svalue, StringComparison.OrdinalIgnoreCase))
-            {
-                return "false";
-            }
-            else if ("null".Equals(svalue, StringComparison.OrdinalIgnoreCase))
-            {
-                return null;
-            }
-
-            return svalue;
```

Comment:
I used Riders refactoring to turn this into a switch expression, and it lost this default case for string. Good I had tests for that.